### PR TITLE
fix changing selected option on click

### PR DESCRIPTION
**Pull request by ee923925github**

### CODE REVIEW

File: src/SelectSingle.vue [hunk 1]
```diff
@@ -131,8 +131,6 @@
 			},
 			updateMenuState(open: boolean, callFocus = true) {
 				this.open = open
-
-				if (!open) this.$emit('select', this.options[this.activeIndex])
```
Comment (ee923925github) [inline src/SelectSingle.vue:135]: it used to cause doubled `select` event

**Review by RED143 — APPROVED** [review]

File: src/SelectSingle.vue [hunk 2]
```diff
@@ -172,6 +170,7 @@
 				}
 			},
 			handleOptionClick(event: MouseEvent, index: number) {
+				this.selectOption(index)
```
Comment (ee923925github) [inline src/SelectSingle.vue:173]: Click on option doesn't change `selectedIndex` and `inputValue` vars.
ezgif com-gif-maker

**Review by wickkidd — APPROVED** [review]

File: src/SelectSingle.vue (hunk 2 above)
Comment (jackkoppa) [reply src/SelectSingle.vue:173]: Hi @ee923925github - I'm realizing I don't understand how this is working right now 🤔 

How was it working *without* this line? What was causing the value to be updated? I guess that `this.$emit('select')` would be called in `updateMenuState`? And that would trigger the update of the `v-model`?

Comment (jackkoppa) [reply src/SelectSingle.vue:173]: I think that makes sense. Let's do it; thanks for investigating this!

**Review by jackkoppa — APPROVED** [review]
Lezz do it!

File: src/SelectSingle.vue (hunk 2 above)
Comment (ee923925github) [reply src/SelectSingle.vue:173]: yes, it worked because of emitted event `select`. But component's variables were not updated.

**Pull request merged**